client/helpers.go

```diff
@@ -158,21 +141,16 @@ func changeTargetMeta(repo *tuf.Repo, c changelist.Change) error {
 		}
 		files := data.Files{c.Path(): *meta}
 
-		err = doWithRoleFallback(c.Scope(), func(role string) error {
-			_, e := repo.AddTargets(role, files)
-			return e
-		})
-		if err != nil {
+		// Attempt to add the target to this role, with no fallback
```

Comment:
Non-blocking nit:  same here - would it make sense to just leave out the idea of fallback?